frontend/device/android/device.lua

```diff
@@ -375,6 +377,21 @@ function Device:canExecuteScript(file)
     end
 end
 
+function Device:lightDialog()
+    local usleep = require("ffi/util").usleep
+    local title = android.isEink() and _("Frontlight settings") or _("Light settings")
+    android.settings.dialog(title, _("Brightness"), _("Warmth"), _("Ok"))
+    repeat
+        usleep(75000) -- sleep 75ms before next check if dialog was quit
```

Comment:
What's the rationale behind this value?

Reply:
It blocks the Lua VM while the dialog is shown. It should be a number at least x10-20 the time spent checking the condition (~1ms) and lower enough, so the user cannot perceive it as a UI block once it press the ok button. 

I think any value from 10 to 250ms is fine

Reply:
If 10 is perfectly fine then it sounds like doubling or tripling to 20 or 30 for safety should be plenty. :-)